utils/enums.py

```diff
@@ -0,0 +1,12 @@
+from enum import Enum
+
+class UserStatus(str, Enum):
+    
+    TO_CONTACT = "To contact"
+    IN_REVIEW = "In review"
+    INTERVIEWED = "Interviewed"
+    ACCEPTED = "Accepted"
+    NO_SHOW = "No show"
+    REJECTED = "Rejected"
+    TO_BE_ONBOARDED = "To be onboarded"
+    CONTACTED = "Contacted"
```

Comment:
Would it be a problem if you made the enum strings uppercase as well? eg; TO_CONTACT = "TO_CONTACT"